docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/05 - Visits.md

```diff
@@ -0,0 +1,44 @@
+# Visits
+
+## Overview
+The **Visit** designates the instance in time where a candidate has visited or contacted a site and underwent any sort of data collection.
+
+**Visit** and **Timepoint** are terms used interchangeably in LORIS. **Visits** are defined in the `Visit_Windows` table of the database.
```

Comment:
Is there any chance we could change all mentions of **visit** to **timepoint** for consistency?  Is there any particular reason to be using both terms (other than we switched at some point and didn't change all) ?

Some projects (ex. one of CAP's data contributors) are creating **timepoints** just for surveys, which makes **visit** misleading and potentially confusing.  And there may be other different milestones used to classify a data collection; **timepoints** is more flexible.

_(Side Note: Perhaps the practice of creating separate timepoints for surveys could be resolved in a different way, like frontend labels/separations of instrument type within **behavioural forms** as discussed @ridz1208 ?  Although they may need to be able to set the windows according to a different **timepoint date**... TBD)_